shiny/playwright/controller/_accordion.py

```diff
@@ -307,6 +307,55 @@ def set(
                 )
             self.accordion_panel(elem_value).set(elem_value in open, timeout=timeout)
 
+    def expect_class(
+        self,
+        class_name: str,
+        has_class: bool,
```

Comment:
I'm on the fence of whether or not to include this parameter. 

If we keep it, I'm thinking we make it a kwarg and default it to `True`. As that'd probably be the common case.